starknet_py/common.py

```diff
@@ -1,12 +1,11 @@
 from typing import List, Literal, Optional, Union
 
-from starkware.starknet.services.api.contract_class import ContractClass
-
 from starknet_py.compile.compiler import (
     Compiler,
     StarknetCompilationSource,
     create_contract_class,
```

Comment:
Let's make sure this branch is rebased on recent development changes properly. `create_contract_class` is now defined in `common.py` as compiler module is deprecated (they're duplicated for now).